policies.go

```diff
@@ -862,10 +870,11 @@ func (d *dcAwareRR) AddHost(host *HostInfo) {
 }
 
 func (d *dcAwareRR) RemoveHost(host *HostInfo) {
+	connAddr, _ := host.ConnectAddress()
```

Comment:
just return if unable to get conn addr but also need to add a check for this in `AddHost` so hosts with invalid connect addresses are not added